pkg/controller/deliverable/reconciler.go

```diff
@@ -90,7 +91,16 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		case realizer.ApplyStampedObjectError:
 			r.conditionManager.AddPositive(TemplateRejectedByAPIServerCondition(typedErr))
 		case realizer.RetrieveOutputError:
-			r.conditionManager.AddPositive(MissingValueAtPathCondition(typedErr.ResourceName(), typedErr.JsonPathExpression()))
+			switch typedErr.Err.(type) {
```

Comment:
I'm not sold on this, but I think we can work on this outside of this PR (my next PR for handing errors maybe?)

Reply:
Sounds good